mathml/elements/mfrac.json

```
@@ -190,6 +190,54 @@
             }
           }
         },
+        "linethickness": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": false
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": "1"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": false
+              },
+              "opera_android": {
+                "version_added": false
+              },
+              "safari": {
+                "version_added": "5.1",
+                "notes": "Does not support <length> values before version 9, and support only `0`, `thin`, `medium`, and `thick`."
```

Some formatting and wording suggestions here:

```suggestion
                "notes": "Before Safari 9, &lt;length&gt; values are not supported. Only <code>0</code>, <code>thin</code>, <code>medium</code>, and <code>thick</code> are supported."
```